arch/M68K/M68KDisassembler.c

```diff
@@ -2339,12 +2347,14 @@ static void d68020_extb_32(m68k_info *info)
 
 static void d68000_jmp(m68k_info *info)
 {
+	set_insn_group(info, M68K_GRP_JUMP);
 	cs_m68k* ext = build_init_op(info, M68K_INS_JMP, 1, 0);
```

Comment:
Alright. I will fix. Maybe having c89 flags for the compilers to enforce this then?